estuary_updater/handlers/koji.py

```diff
@@ -96,6 +96,13 @@ def build_handler(self, msg):
                     for component in components:
                         component_build = self.get_or_create_build(component)
                         build.components.connect(component_build)
+            elif build.__label__ == ContainerKojiBuild.__label__:
+                extra_json = msg['body']['msg']['info']['extra']
+                if extra_json.get('typeinfo').get('operator_manifests').get('archive'):
```

Comment:
Similar comment as in https://github.com/release-engineering/estuary-api/pull/271 :-)

+ would be good to add unit test. I can provide fake json message if needed.

Reply:
Thanks @lslebodn! Could you please provide one?